src/AutoRest.CSharp/Common/Output/Models/Serialization/PropertySerialization.cs

```diff
@@ -28,23 +28,16 @@ internal abstract class PropertySerialization
         public CSharpType? SerializedType { get; }
 
         public bool IsRequired { get; }
-        public bool ShouldSkipSerialization { get; }
-        public bool ShouldSkipDeserialization { get; }
+        public bool ShouldExcludeInWireSerialization { get; }
 
-        protected PropertySerialization(string parameterName, TypedValueExpression value, string serializedName, CSharpType? serializedType, bool isRequired, bool shouldSkipSerialization) :
-            this(parameterName, value, serializedName, serializedType, isRequired, shouldSkipSerialization, false)
-        {
-        }
-
-        protected PropertySerialization(string parameterName, TypedValueExpression value, string serializedName, CSharpType? serializedType, bool isRequired, bool shouldSkipSerialization, bool shouldSkipDeserialization)
```

Comment:
we remove this ctor and the property `ShouldSkipDeserialization` because there is never a scenario that we are assigning this property to `true`. It is really not needed anywhere.